README.md

```diff
@@ -27,13 +27,13 @@ You will also need to install:
 
 ### On Unix
 
-   * `python` (`v2.7` recommended, `v3.x.x` is __*not*__ supported)
+   * `python v2.7, v3.5, v3.6, or v3.7`
```

Comment:
This isn't accurate, 3.x won't be supported unless EXPERIMENTAL_NODE_GYP_PYTHON3 is set (and its not documented). I'd revert the README changes entirely, they shouldn't be mixed with Travis config changes anyway.